src/server/modules/user/index.js

```diff
@@ -168,5 +219,52 @@ export default new Feature({
         res.redirect('/profile');
       });
     }
+
+    // Setup Google OAuth
+    if (settings.user.auth.google.enabled) {
+      app.use(passport.initialize());
+
+      app.get(
+        '/auth/google',
+        passport.authenticate('google', {
+          scope: [
+            'https://www.googleapis.com/auth/userinfo.email',
```

Comment:
@verdverm Maybe its better to move these settings into `config/user.js` -> auth -> google ?

Reply:
Done, did the same for the Facebook items.